db/migrate/20200726022620_add_stripe_customer_id_to_users.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class AddStripeCustomerIdToUsers < SolidusSupport::Migration[5.1]
+  def up
+    add_column :spree_users, :stripe_customer_id, :string, unique: true
+
+    Spree::User.includes(bill_address: :country, ship_address: :country).find_each do |u|
+      user_stripe_payment_sources = user&.wallet&.wallet_payment_sources&.select do |wps|
```

Comment:
I think `user` here is meant to be `u` or probably the other way around, isn't it?

Reply:
Good catch, thank you! I changed it to `u`.